commons/src/main/java/de/tinf15b4/kino/data/cinemas/Cinema.java

```diff
@@ -58,7 +67,7 @@ public Cinema(String name, String street, String hnr, String postcode, String ci
     }
 
     public String getAddress() {
-        return String.format("%s %s\n%s %s, %S", street, hnr, postcode, city, country);
+        return String.format("%s %s%n%s %s, %S", street, hnr, postcode, city, country);
```

Comment:
Exactly what you mentioned: The reason is that each platform users the corresponding linebreak